tasks/network_tracer.py

```diff
@@ -0,0 +1,204 @@
+import datetime
+import glob
+import os
+import contextlib
+import shutil
+import tempfile
+
+from invoke import task
+from subprocess import check_output, CalledProcessError
+
+from .utils import bin_name, get_build_flags, REPO_PATH, get_version, get_git_branch_name, get_go_version, get_git_commit
+from .build_tags import get_default_build_tags
+
+BIN_DIR = os.path.join(".", "bin", "network-tracer")
+BIN_PATH = os.path.join(BIN_DIR, bin_name("network-tracer", android=False))
+
+EBPF_BUILDER_IMAGE = 'datadog/tracer-bpf-builder'
+EBPF_BUILDER_FILE = os.path.join(".", "Dockerfiles", "network-tracer", "Dockerfile-ebpf")
+
+BPF_TAG = "linux_bpf"
+
+@task
+def build(ctx, race=False, rebuild_ebpf_builder=False, incremental_build=False, use_embedded_libs=False, puppy=False, use_venv=False):
+    """
+    Build the network_tracer
+    """
+
+    if rebuild_ebpf_builder:
+        build_ebpf_builder(ctx)
+
+    build_object_files(ctx)
+
+    # TODO use pkg/version for this
+    main = "main."
+    ld_vars = {
+        "Version": get_version(ctx),
+        "GoVersion": get_go_version(),
+        "GitBranch": get_git_branch_name(),
+        "GitCommit": get_git_commit(),
+        "BuildDate": datetime.datetime.now().strftime("%FT%T%z"),
+    }
+
+    ldflags, gcflags, env = get_build_flags(
+        ctx, use_embedded_libs=use_embedded_libs, use_venv=use_venv
+    )
+
+    # Add custom ld flags
+    ldflags += ' '.join(["-X '{name}={value}'".format(name=main+key, value=value) for key, value in ld_vars.items()])
+    build_tags = get_default_build_tags(puppy=puppy).append(BPF_TAG)
+
+    # TODO static option
+    cmd = 'go build {race_opt} {build_type} -tags "{go_build_tags}" '
+    cmd += '-o {agent_bin} -gcflags="{gcflags}" -ldflags="{ldflags}" {REPO_PATH}/cmd/network-tracer'
+
+    args = {
+        "race_opt": "-race" if race else "",
+        "build_type": "-i" if incremental_build else "-a",
+        "go_build_tags": " ".join(build_tags),
+        "agent_bin": BIN_PATH,
+        "gcflags": gcflags,
+        "ldflags": ldflags,
+        "REPO_PATH": REPO_PATH,
+    }
+
+    ctx.run(cmd.format(**args), env=env)
+
+
+@task
+def test(ctx):
+    """
+    Run tests on eBPF parts
```

Comment:
Do these tests run in an environment where we can run the actual eBPF code?

Reply:
Not yet but this is planned on the next PR